kythe/go/indexer/indexer.go

```diff
@@ -853,6 +899,20 @@ func matchesBuildTags(fpath string, data []byte, bc *build.Context) bool {
 	return err == nil && match
 }
 
+// unmarshalProtoBase64 parses a base64 encoded proto string into the supplied
+// message, mutating msg. If the string cannot be unmarshalled, the function
```

Comment:
The "If the string cannot be unmarshalled, the function returns an error" bit is extraneous